mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/api/akka/route/v1/ResourceClustersNonLeaderRedirectRoute.java

```diff
@@ -263,7 +263,11 @@ private Route getTaskExecutorState(ClusterID clusterID, TaskExecutorID taskExecu
     private Route disableTaskExecutors(ClusterID clusterID) {
         return entity(Jackson.unmarshaller(DisableTaskExecutorsRequest.class), request -> {
             log.info("POST /api/v1/resourceClusters/{}/disableTaskExecutors called with body {}", clusterID, request);
-            return withFuture(gateway.getClusterFor(clusterID).disableTaskExecutorsFor(request.getAttributes(), Instant.now().plus(Duration.ofHours(24))));
+            return withFuture(gateway.getClusterFor(clusterID).disableTaskExecutorsFor(
+                request.getAttributes(),
+                request.getExpirationDurationInHours() == null || request.getExpirationDurationInHours() <= 0L ?
```

Comment:
If it's nullable, then we should capture this in the type. I would prefer that this is not nullable, btw since this is under our control.

Reply:
ok will add nonnull annotation to the request.